frontend/src/features/properties/components/forms/subforms/PagedBuildingForms.tsx

```diff
@@ -58,6 +73,22 @@ const PagedBuildingForms: React.FC<PagedBuildingFormsProps> = (props: PagedBuild
         name="buildings"
         render={arrayHelpers => (
           <div>
+            {showDelete && (
+              <GenericModal
+                display={showDelete}
+                cancelButtonText="Close"
+                okButtonText="Delete"
+                handleOk={() => {
+                  deletePagedBuilding(buildings, currentPage, setCurrentPage, arrayHelpers);
```

Comment:
This may be outside the scope of this story, but it would be nice if the modal tried to submit the form whenever a building is deleted. The problem with this is that you would have to check for other validation errors in the form and display an error to prevent situations where an auto-submitted deleted building fails.

Reply:
ahh yes that's a good idea. right now it might surprise a few users when they don't hit submit after removing... the modal 'feels' like a submit. maybe for now I should add a message saying finalize by submitting the form in the meantime...